.github/workflows/windows_intel_oneAPI.yml

```diff
@@ -0,0 +1,102 @@
+name: Windows tests (MSVC + ifx + OpenBLAS)
+# TODO: replace OpenBLAS with MKL. This is harder to get to build, so we merged with OpenBLAS first.
```

Comment:
Now I am beginning my work on this. First step - https://github.com/scipy/scipy/issues/20728